ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py

```diff
@@ -456,3 +456,45 @@ def test_torch_swapaxes(
         axis0=axis0,
         axis1=axis1,
     )
+
+
+#chunk
+@handle_cmd_line_args
+@given(
+    dtype_value_shape=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(
+                set(ivy_torch.valid_float_dtypes)),
+        ),
+        num_arrays=st.shared(helpers.ints(min_value=1), key="num_arrays"),
+        shape=st.shared(helpers.get_shape(min_num_dims=1), key="shape"),
+    ),
+    dim=helpers.get_axis(
+        shape=st.shared(helpers.get_shape(min_num_dims=1), key="shape"),
+    ).filter(lambda axis: isinstance(axis, int)),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.torch.chunk"
+    ), 
+)
+def test_torch_chunk(
+    dtype_value_shape,
+    dim,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    fw,
+):
+    input_dtype,value = dtype_value_shape
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend='torch',
+        fn_tree='chunk',
+        input=np.asarray(value,dtype=input_dtype),
+        dim=dim,
```

Comment:
please generate data for the `chunks` parameter and add the argument here

Reply:
Hi , thanks for the review , how do I generate data for the parameter ?, I'm a little lost on that

Reply:
through the helpers function, just like how you did for the input arrays with `helpers.dtype_and_values`, more can be found in the documentation!
https://lets-unify.ai/ivy/deep_dive/16_ivy_frontends_tests.html